build.sbt

```diff
@@ -9,7 +9,7 @@ lazy val root = (project withId "stryker4s" in file("."))
     // Publish locally for sbt plugin testing
     addCommandAlias(
       "publishPluginLocal",
-      "set ThisBuild / version := \"0.0.0-TEST-SNAPSHOT\"; stryker4s-core2_12/publishLocal; stryker4s-api2_12/publishLocal; sbt-stryker4s2_12/publishLocal; stryker4s-api/publishLocal; sbt-stryker4s-testrunner/publishLocal"
+      "set ThisBuild / version := \"0.0.0-TEST-SNAPSHOT\"; stryker4s-core2_12/publishLocal; stryker4s-api2_12/publishLocal; sbt-stryker4s2_12/publishLocal; stryker4s-api/publishLocal; sbt-stryker4s-testrunner/publishLocal; stryker4s-command-runner/publishLocal; stryker4s-command-runner2_12/publishLocal"
```

Comment:
I think this should be a separate command. There is already a separate one for Maven, so it would make sense to have one for the command-runner too. Then only publishing 2.13 versions should also be enough (it doesn't matter what your own Scala version is for the command-runner, so you don't need 2.12):

```
stryker4s-core/publishLocal; stryker4s-api/publishLocal; stryker4s-command-runner/publishLocal
```

Reply:
Done.